### ISSUE TITLE
Failed to install vsphere plugin

### ISSUE THREAD

**Issue by sweetpeaito** [terraform-providers/terraform-provider-vsphere#936]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform
issue trackers reserved for bug reports and feature requests. For general usage
questions, please see: https://www.terraform.io/community.html.

### Terraform Configuration Files

```hcl
# provider "vSphere"{
    user="xxxx"
    password="xxxx"
    vsphere_server="xxxx"

    all_unverified_ssl=true
}

### Debug Output


### Panic Output


### Expected Behavior

What should have happened?

### Actual Behavior

What actually happened?
Error installing provider "vSphere": failed to find installed plugin version 1.14.0; this is a bug in Terraform and should be reported.

Terraform analyses the configuration and state and automatically downloads
plugins for the providers used. However, when attempting to download this
plugin an unexpected error occurred.

This may be caused if for some reason Terraform is unable to reach the
plugin repository. The repository may be unreachable if access is blocked
by a firewall.

If automatic installation is not possible or desirable in your environment,
you may alternatively manually install plugins by downloading a suitable
distribution package and placing the plugin's executable file in the
following directory:
    terraform.d/plugins/windows_amd64


Error: failed to find installed plugin version 1.14.0; this is a bug in Terraform and should be reported
### Steps to Reproduce

Please list the steps required to reproduce the issue, for example:
1. `terraform init`

**Comment by bill-rich** [comment]
Hi @sweetpeaito! 

It looks like your running into an issue with being able to download the provider executable. Please make sure you're able to download the [provider](https://releases.hashicorp.com/terraform-provider-vsphere/1.14.0/) from the system you're running Terraform on. If that works, check to make sure your firewall is not blocking Terraform.

**Comment by sweetpeaito** [comment]
Hi Bill,
Thank you for your response . I tried downloading it directly via the link you provided and i downloaded it. I tried running it again but got this error :


I have tried putting the file manually in the path specified. 
kip kool 

    On Wednesday, January 15, 2020, 02:26:35 AM GMT+1, Bill Rich <notifications@github.com> wrote:  
 
 
Closed #936.
